app/templates/programs.hbs

```diff
@@ -41,6 +41,12 @@
         {{t 'general.savedSuccessfully'}}
       </div>
     {{/if}}
+	
+    {{#if deleted}}
+      <div class='saved-program'>
+        {{fa-icon 'external-link-square'}} {{savedProgram.title}} {{t 'general.removedSuccessfully'}}
```

Comment:
Please remove the fa-icon as we use the square to indicate a link to another resource.